.eslintrc

```diff
@@ -20,7 +20,8 @@
       "prefer-reflect": 2,
       "no-warning-comments": [1, { "terms": ["todo", "fixme"], "location": "start" }],
       "react/sort-comp": 0,
-      "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+      "react/require-extension": "off",
+      "import/extensions": [1, { "js": "never", "jsx": "never" }],
```

Comment:
Fix pushed, let me know if I should squash or amend the original instead based on the Times' preference.